pandas/tests/frame/test_analytics.py

```diff
@@ -1004,6 +1013,29 @@ def test_sum_prod_nanops(self, method, unit):
         expected = pd.Series(result, index=['A', 'B'])
         tm.assert_series_equal(result, expected)
 
+        if method == 'sum':
```

Comment:
*maybe* split the sum to a separate test (I know you have to repeat the parameterization), but I think its ok / worth it.